JavaFiddle-ejb/src/test/java/UserBeanTest.java

```diff
@@ -0,0 +1,167 @@
+import static org.junit.Assert.*;
+
+import org.eclipse.persistence.config.PersistenceUnitProperties;
+import org.eclipse.persistence.platform.database.SybasePlatform;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import ru.javafiddle.core.ejb.*;
+import ru.javafiddle.jpa.entity.*;
+
+import javax.ejb.embeddable.EJBContainer;
+import javax.naming.Context;
+import javax.naming.InitialContext;
+import javax.naming.NamingException;
+import java.io.File;
+import java.io.IOException;
+import java.io.UnsupportedEncodingException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.security.NoSuchAlgorithmException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class UserBeanTest {
+
+    private static final Logger logger =
```

Comment:
Have you configured logging for your tests? Remember it's the client-side code.
Actually writing to System.out will be enough for most of the cases.